doc/changelog/06-Ltac2-language/16859-ltac2_parse_arrays.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Support for parsing Ltac2 array literals ``[| ... |]``
```

Comment:
also mention list literals

Reply:
I didn't add list literals, I just had to pull them out into a separate nonterminal to get it to work, but there's no user-visible change w.r.t. lists.